CHANGELOG.md

```diff
@@ -27,26 +22,42 @@
 - A sad goodbye [#5873](https://github.com/habitat-sh/habitat/pull/5873) ([elliott-davis](https://github.com/elliott-davis))
 - Update version [#5880](https://github.com/habitat-sh/habitat/pull/5880) ([smacfarlane](https://github.com/smacfarlane))
 - Add 0.69.0 release post [#5877](https://github.com/habitat-sh/habitat/pull/5877) ([smacfarlane](https://github.com/smacfarlane))
+
+#### New Features & Enhancements
+- Add simple auth and TLS to supervisor HTTP gateway [#5874](https://github.com/habitat-sh/habitat/pull/5874) ([raskchanky](https://github.com/raskchanky))
+
+## [0.69.0](https://github.com/habitat-sh/habitat/tree/0.69.0) (2018-11-27)
```

Comment:
The issues here don't line up with https://github.com/habitat-sh/habitat/blob/0.69.0/CHANGELOG.md, which is what I'd expect to see. Some 0.70.0 issues are down here.

Reply:
Hmm strange, i got that list from the `git log --pretty=oneline --merges 0.68.0..0.69.0` I'll correct these momentarily.